ray-curator/ray_curator/stages/audio/io/audio_reader.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+from dataclasses import dataclass
+
+from loguru import logger
+
+from ray_curator.stages.base import ProcessingStage
+from ray_curator.tasks import DataEntry, _EmptyTask
+from ray_curator.utils.file_utils import get_all_files_paths_under
+
+
+@dataclass
+class AudioReaderStage(ProcessingStage[_EmptyTask, DataEntry]):
+    """Stage that reads video files from storage and extracts metadata."""
+
+    input_audio_path: str
+    audio_limit: int = -1
```

Comment:
```suggestion
    audio_file_limit: int = -1
```
or `file_limit`?

Reply:
The name is inspired by VideoReader class: 
https://github.com/NVIDIA-NeMo/Curator/blob/ray-api/ray-curator/ray_curator/stages/video/io/video_reader.py#L235
Should it be consistent?